test/server/enrollment-system.spec.js

```diff
@@ -0,0 +1,33 @@
+const chai = require('chai');
+chai.should();
```

Comment:
I've been pushing to insert a newline between 3rd party includes and local ones.